build.gradle.kts

```diff
@@ -11,6 +8,11 @@ plugins {
     id("com.gradle.build-scan").version("2.4.2")
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_9
+    targetCompatibility = JavaVersion.VERSION_1_9
```

Comment:
This breaks source compatibility. Ways out:
- major release
- extract to a separate lib

Reply:
I'd opt for major release